### DISCUSSION TITLE
Color palette not persistent?

### DISCUSSION THREAD

**Discussion by sensor-freak** [arendst/Tasmota#11218] [Q&A] [answered]
Hello,

it seems to me that the color palette is not persisted. Each time I restart the device, the palette is empty.

Is this a known behavior? Is this behavior wanted? Or is this a bug?

Thanks for any hint!

**Comment by barbudor** [comment, marked as answer]
Indeed, `Palette` is not saved to flash.
You need to use a boot-time rule such as `on system#boot do Palette xxxxx endon` to set it back at each restart.

**Reply by sensor-freak** [reply]
Thank you, it works that way!
